### DISCUSSION TITLE
Cannot invoke plugins with semantic kernel

### DISCUSSION THREAD

**Discussion by akordowski** [microsoft/semantic-kernel#6557] [Q&A] [answered]
I am trying to invoke plugins with semantic kernel following the available examples. But the responses indicate that the plugin is never executed and the break points also are not triggered. I have access only to the `gpt-35-turbo` version `0301` model. Does the model don't support the plugin execution? I couldn't find any information on that.

**Comment by dmytrostruk** [comment, marked as answer]
@akordowski `gpt-35-turbo` version `0301` does not support function calling. Here is a list of supported models:
https://platform.openai.com/docs/guides/function-calling/supported-models
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/fde944ac-31fb-4af6-8f49-c97ef74d77e1)

**Reply by akordowski** [reply]
Thank you for the information!
